package.json

```diff
@@ -39,10 +39,7 @@
     "npm": ">=6"
   },
   "files": [
-    "src",
-    "index.js",
-    "index.d.ts",
```

Comment:
dont we still want this file to be part of the npm package?

Reply:
It will be generated in dist